README.md

```diff
@@ -20,77 +20,57 @@
 <!-- TODO: code coverage -->
 </p>
 
-# Overview
-
-Versatile Data Kit (VDK) is an open source framework that enables anyone with basic SQL or Python knowledge to build their own data pipelines.
-
-Versatile Data Kit enables Data Engineers to develop, deploy, run and manage Data Jobs. **A Data Job is a data processing workload** and can be written in Python, SQL, or both at the same time. A Data Job enables Data Engineers to implement automated pull ingestion (E in ELT) and batch data transformation (T in ELT) into a database or any type of data storage.
 
-![Without / With Versatile Data Kit](./support/images/versatile-data-kit-before-after-light.svg#gh-light-mode-only)
-![Without / With Versatile Data Kit](./support/images/versatile-data-kit-before-after-dark.svg#gh-dark-mode-only)
-
-Versatile Data Kit consists of two main components:
+# Overview
+Versatile Data Kit (VDK) is an open source framework that enables anyone with basic SQL or Python knowledge to build and execute their own data workflows.
```

Comment:
Instead of "build and execute" let's use 'build, run, and manage' as it's the one we use in the "about" info

Reply:
updated